src/main/java/org/waarp/openr66/dao/database/mariadb/MariaDBTransferDAO.java

```diff
@@ -0,0 +1,46 @@
+package org.waarp.openr66.dao.database;
```

Comment:
Wrong package (missing mariadb) and therefore need `import org.waarp.openr66.dao.database.DBTransferDAO;`